tests/CssToInlineStylesTest.php

```diff
@@ -266,41 +266,41 @@ public function testSelfClosingTags(): void
 
     public function testConversionAsciiRegular(): void
     {
-        $html = '~';
+        $html = '<p>~</p>';
         $css = '';
-        $expected = "<p>{$html}</p>";
+        $expected = "{$html}";
```

Comment:
```suggestion
        $expected = $html;
```

No need to use string interpolation with the variable as the only content.